install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1699,13 +1704,204 @@ dashboards:
+
+  securityContext:
+    # -- User to run the pod with
+    # runAsUser: 0
+    # -- Run the pod with elevated privileges
+    privileged: false
+    # -- SELinux options for the `cilium-envoy` container
+    seLinuxOptions:
+      level: 's0'
+      # Running with spc_t since we have removed the privileged mode.
+      # Users can change it to a different type as long as they have the
+      # type available on the system.
+      type: 'spc_t'
+    capabilities:
+      # -- Capabilities for the `cilium-envoy` container
+      envoy:
+        # Used since cilium proxy uses setting IPPROTO_IP/IP_TRANSPARENT
+        - NET_ADMIN
+        # We need it for now but might not need it for >= 5.11 specially
+        # for the 'SYS_RESOURCE'.
+        # In >= 5.8 there's already BPF and PERMON capabilities
+        - SYS_ADMIN
+        # Both PERFMON and BPF requires kernel 5.8, container runtime
+        # cri-o >= v1.22.0 or containerd >= v1.5.0.
+        # If available, SYS_ADMIN can be removed.
+        #- PERFMON
+        #- BPF
+
+  # -- Affinity for cilium-envoy.
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - topologyKey: kubernetes.io/hostname
+        labelSelector:
+          matchLabels:
+            k8s-app: cilium-envoy
+
+  # -- Node selector for cilium-envoy.
+  nodeSelector:
+    kubernetes.io/os: linux
+
+  # -- Node tolerations for envoy scheduling to nodes with taints
+  # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+  tolerations:
+  - operator: Exists
+    # - key: "key"
+    #   operator: "Equal|Exists"
+    #   value: "value"
+    #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+
+  # -- The priority class to use for cilium-envoy.
+  priorityClassName: ~
+
+  # -- DNS policy for Cilium envoy pods.
+  # Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+  dnsPolicy: ""
+
+  prometheus:
+    enabled: true
+    serviceMonitor:
+      # -- Enable service monitors.
+      # This requires the prometheus CRDs to be available (see https://github.com/prometheus-operator/prometheus-operator/blob/main/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml)
+      enabled: false
+      # -- Labels to add to ServiceMonitor cilium-envoy
+      labels: {}
+      # -- Annotations to add to ServiceMonitor cilium-envoy
+      annotations: {}
+      # -- Interval for scrape metrics.
+      interval: "10s"
+      # -- Specify the Kubernetes namespace where Prometheus expects to find
+      # service monitors configured.
+      # namespace: ""
+      # -- Relabeling configs for the ServiceMonitor cilium-envoy
+      relabelings:
+        - sourceLabels:
+            - __meta_kubernetes_pod_node_name
+          targetLabel: node
+          replacement: ${1}
+      # -- Metrics relabeling configs for the ServiceMonitor cilium-envoy
+      metricRelabelings: ~
+    port: "9964"
```

Comment:
Hmm, shouldn't this be left empty and the values from `envoy.prometheus.port` used?